tools/zbjammer

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python
+
+"""
+zbjammer: enable Zigbee jamming (firmware option)
```

Comment:
I'm thinking we should add a brief thing here like "You are responsible for using this in accordance with local laws and regulations." or something.

Reply:
I don't disagree - feel free to add/edit as you see fit! :)

Reply:
Done!

Reply:
Sorry I didn't get to it first!